docs/manual/installation.rst

```diff
@@ -31,10 +31,10 @@ Then install phpspec with the composer install command:
 
     $ composer install
 
-Follow instructions on `the composer website <https://getcomposer.org/download/`_
+Follow instructions on `the composer website <https://getcomposer.org/download/>`_
 if you don't have it installed yet.
 
-Phpspec with its dependencies will be installed inside the ``vendor`` folder
+phpspec with its dependencies will be installed inside the ``vendor`` folder
```

Comment:
Yes, but `phpspec` is a name & most of the time it's used in lowercase letters (see the beginning of the file for example)